src/components/Collapse.js

```diff
@@ -0,0 +1,35 @@
+import _ from 'lodash'
+import { div, span } from 'react-hyperscript-helpers'
+import { icon } from 'src/components/icons'
+import * as Style from 'src/libs/style'
+import { Component } from 'src/libs/wrapped-components'
+
+
+export default class Collapse extends Component {
+  constructor(props) {
+    super(props)
+    this.state = { visible: !props.defaultHidden }
+  }
+
+  render() {
+    const { title, titleExpand } = this.props
+    const { visible } = this.state
+
+    return div(_.omit(this.props, ['defaultHidden', 'title', 'titleExpand']), [
+      div({ style: { marginBottom: '0.5rem' } }, [
+        div(
+          {
+            style: { display: 'flex', cursor: 'pointer', alignItems: 'center' },
+            onClick: () => this.setState({ visible: !visible })
+          },
+          [
+            span({ style: { color: Style.colors.secondary } }, title),
+            icon(visible ? 'angle down' : 'angle left', { style: { marginLeft: '0.25rem' } })
```

Comment:
Open question: do we like this design for the collapse icon? Seems more natural for the icon to go on the left, but then in FC we ran into the issue where that pushed the title to the right and it didn't line up with the form.

Reply:
Personally, I think it looks great there.